### PR TITLE
Add `setPRNG` to the set of bindings

### PR DESCRIPTION
**Pull request by athanclark**
This would let people who work with react-native to come up with their own random number generator

### CODE REVIEW

**Comment by athanclark** [comment]
@throughnothing ping..? :(

File: src/Crypt/NaCl/Random.purs [hunk 1]
```diff
@@ -1,8 +1,13 @@
 module Crypt.NaCl.Random where
 
+import Data.Unit (Unit)
+import Data.ArrayBuffer.Types (Uint8Array)
 import Control.Monad.Eff (Eff)
 
 import Crypt.NaCl.Types (Nonce, NACL_RANDOM)
 
+-- | Assigns a new PRNG for all random data generation
+foreign import setPRNG :: forall e e1. (Uint8Array -> Int -> Eff e1 Unit) -> Eff (naclRandom :: NACL_RANDOM | e) Unit
```
Comment (throughnothing) [inline src/Crypt/NaCl/Random.purs:10]: Should the function passed in also have the `NACL_RANDOM` effect?

**Comment by throughnothing** [comment]
@athanclark Sorry, I must've missed the notification on the initial PR, apologies!  Thanks for submitting this :)

**Comment by athanclark** [comment]
Hey! Sorry for being nitpicky :)

But, I argue that it shouldn't need the effect token, because the code being invoked doesn't necessarilly cause a NACL-specific effect, but if you think it should be required, I wouldn't mind.

**Comment by throughnothing** [comment]
I think that makes sense.

**Pull request merged**